app/views/playbook/index.html.slim

```diff
@@ -0,0 +1,66 @@
+doctype html
+html lang="#{I18n.locale}"
+
+  = render 'head'
+
+  body
+    .site-wrap
+      = render 'shared/usa_banner'
+
+      nav.bg-white
+        .container.cntnr-wide.px2.py3
+          = render 'nav_top'
+          .clearfix.mb3
+            = image_tag(asset_url('playbook/graphic-circle.png'), alt: APP_NAME, width: 240,
+              class: 'ml6 right sm-show')
+            p.mt3.h4.overflow-hidden.serif.line-height-3.pb-lead
+              | “Identity management,” or making sure that only the correct people get access
+                to systems, is central to delivering public services online. We are sharing
+                these principles to help developers and technologists in government agencies
+                build usable, secure, and privacy-protecting consumer identity management
+                systems.
+
+      .pb-bg-navy
+        .container.cntnr-wide.px2.py5
+          h2.mt0.mb4.h1.white The principles
+          .clearfix.mxn1
+            .sm-col.sm-col-6.mb2.px1
+              = link_to(playbook_principles_path(anchor: 'pb-users'),
+                class: 'py3 pl3 pr5 bg-white block pb-principle-cntnr') do
+                .navy.pb-principle
+                  .table-cell.align-middle.center
+                    = image_tag(asset_url('playbook/graphic-users.svg'), alt: APP_NAME)
+                  .table-cell.sm-pl3.pl2.align-middle.bold Focus on user needs
```

Comment:
see above for potential refactor of this section; in general, we should standardize to have mobile first classes before larger viewport overrides (i.e., `pl2 sm-pl3`) -- i think it reads a bit better ("this has padding left of 2 units in general, and if screens are big, it becomes 3 units")